source/agora/consensus/data/Params.d

```diff
@@ -114,6 +114,8 @@ public immutable class ConsensusParams
 /// Ditto
 public struct ConsensusConfig
 {
+    /// This will be set to the initial time of Coinnet blockchain
+    /// (It should NEVER be overidden except in the case of testing)
```

Comment:
I mean, that's in the definition of the struct

Reply:
Removed